event_attribution_reporting.md

```diff
@@ -35,7 +35,7 @@ The browser will expose a new interface:
 
 ```
 [Exposed=Window] interface AttributionReporting {
-  bool registerAttributionSource(AttributionSourceParams params);
+  Promise<void> registerAttributionSource(AttributionSourceParams params);
```

Comment:
I'm not sure if this should be spelled `Promise<void>` or `Promise<undefined>`.

Reply:
I think void has been replaced by undefined in WebIDL from what I can see
https://heycam.github.io/webidl/#idl-undefined

Reply:
Done.